map.js

```diff
@@ -33,7 +34,8 @@ Output:
 ]*/
 
 export function makeArrayWithIsHungry(arr) {
-    return [];
+    arr.map(object => object.isHungry = true);
```

Comment:
I know this passing the test but I'm not certain is good practice. Either set the returned array to a variable or have a return for the arr.map() is probably the best way to do this.